spec/widgets/category/category-widget-model.spec.js

```diff
@@ -32,6 +33,23 @@ describe('widgets/category/category-widget-model', function () {
     });
   });
 
+  describe('search state', function () {
```

Comment:
It looks like we're not really testing the CategoryWidgetModel in these tests. Instead, it looks like are we're testing is the CategoryDataviewModel, so these should probably be moved to CartoDB.js.